src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -1077,4 +1077,13 @@ Do you want to proceed?</value>
   <data name="All_methods" xml:space="preserve">
     <value>All methods</value>
   </data>
+  <data name="Inline_Errors_Error" xml:space="preserve">
+    <value>Inline Errors - Error</value>
+  </data>
+  <data name="Inline_Errors_Rude_Edit" xml:space="preserve">
+    <value>Inline Errors - Rude Edit</value>
+  </data>
+  <data name="Inline_Errors_Warning" xml:space="preserve">
+    <value>Inline Errors - Warning</value>
```

Comment:
`Inline Diagnostics`?  that way you don't have the wierd `Errors - Warning` confusion?